spec/models/export/relation_spec.rb

```diff
@@ -91,19 +92,79 @@
     end
 
     it 'writes a header for the submissions export' do
-      expect(output_lines.first).to eql(
-        'TaskID,SubmissionID,Status,SubmissionType,SubmissionFileType,ContractEntryCount,' \
-        'ContractValue,InvoiceEntryCount,InvoiceValue,CCSManagementChargeValue,CCSManagementChargeRate,' \
-        'CreatedDate,CreatedBy,SupplierApprovedDate,SupplierApprovedBy,FinanceExportDate,PONumber'
-      )
+      expect(output_lines.first).to eql(%w[
+        TaskID
+        SubmissionID
+        TotalSpend
+        Status
+        SubmissionType
+        SubmissionFileType
+        ContractEntryCount
+        ContractValue
+        CCSManagementChargeValue
+        CCSManagementChargeRate
+        CreatedDate
+        CreatedBy
+        SupplierApprovedDate
+        SupplierApprovedBy
+        FinanceExportDate
+        PONumber
+      ].join(','))
     end
 
     it 'writes each submission to the export' do
       expect(output_lines.length).to eql(3)
-      expect(output_lines.find { |line| line.match('PO1234') }).to eql(
-        "#{submission.task.id},#{submission.id},supplier_accepted,file,xls,1,804.00,1,179.12,1.79," \
-        '#REMOVED,2018-09-18T14:20:35Z,CrForename CrSurname,2018-09-20T10:11:12Z,SubForename SubSurname,,PO1234'
-      )
+
+      submission_record = CSV.parse(output_lines.join("\n"), headers: true).find do |row|
+        row['SubmissionType'] == 'file'
+      end
+
+      aggregate_failures do
+        expect(submission_record.fetch('TaskID'))
+          .to eq(submission.task.id)
+
+        expect(submission_record.fetch('SubmissionID'))
+          .to eq(submission.id)
+
```

Comment:
Would it be prudent to verify that expected value of `TotalSpend` is actually included in the report? e.g.

```
expect(submission_record.fetch('TotalSpend'))
  .to eq(123.45)

```

?

Reply:
Good call, I missed that test!